hbase-oss/src/test/java/org/apache/hadoop/hbase/oss/contract/TestHBOSSContractOpen.java

```diff
@@ -31,6 +33,14 @@ protected Configuration createConfiguration() {
 
   @Override
   protected AbstractFSContract createContract(Configuration conf) {
-    return new HBOSSContract(conf);
+    HBOSSContract contract = new HBOSSContract(conf);
+    try {
+      TestUtils.getFileSystem(conf);
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
Do not use `e.printStackTrace()`.

Reply:
Removed within the overall refactoring, but noticed this is spread around other points in hboss tests. Might need an additional jira for cleaning this out.